### PR TITLE
[Labs] Fixes QueryList onKeyDown only fires for up and down keys (#1403)

### PR DESCRIPTION
**Pull request by paulpooch**
#### Fixes https://github.com/palantir/blueprint/issues/1403

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @paulpooch! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

**Comment by paulpooch** [comment]
Would have ripped out the whole default case but a lot of peoples' linters will freak out...

File: packages/labs/src/components/query-list/queryList.tsx [hunk 1]
```diff
@@ -254,7 +254,7 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
                 event.preventDefault();
                 this.moveActiveIndex(1);
                 break;
-            default: return;
+            default: break;
```
Comment (giladgray) [inline packages/labs/src/components/query-list/queryList.tsx]: yep this is what I expected, given the linter

**Comment by adidahiya** [comment]
thanks @paulpooch!

**Pull request merged**